Intersect.Server/Entities/Npc.cs

```diff
@@ -1633,6 +1633,24 @@ public override EntityPacket EntityPacket(EntityPacket packet = null, Player for
             return pkt;
         }
 
+        public override void OnNearEntityDies(Entity en)
+        {
+            if (en.GetEntityType() == EntityType.Player)
+            {
+                RemoveFromDamageMap(en);
+            }
+        }
+
+        public override void OnNearEntityAttacked(Entity victim, Entity attacker) 
+        {
+            if (Target == null &&
+                IsAllyOf(victim) &&
+                InRangeOf(this, Base.SightRange) &&
```

Comment:
This InRangeOf call seems wrong.

Before, it was calling InRangeOf on the Player, and passing in the NPC. Now it's InRangeOf on the NPC, and passing in the NPC? The NPC will always be in its own sight range though.